.gitignore

```diff
@@ -31,4 +31,6 @@ devnet-consts.json
 /ethereum/cache/
 sui.log.*
 sui/examples/wrapped_coin
-*.prof
\ No newline at end of file
+*.prof
+# Only used for internal testing
+*.xlabs.yaml
```

Comment:
maybe there was a more generic name for this, but 🤷

Reply:
Yeah, maybe `.testnet.yaml` would be better?